ui/app/routes/vault/cluster/settings/configure-secret-backend/section.js

```diff
@@ -2,21 +2,28 @@ import AdapterError from '@ember-data/adapter/error';
 import { set } from '@ember/object';
 import Route from '@ember/routing/route';
 
+// ARG TODO glimmerize
 const SECTIONS_FOR_TYPE = {
   pki: ['cert', 'urls', 'crl', 'tidy'],
 };
 export default Route.extend({
   fetchModel() {
     const { section_name: sectionName } = this.paramsFor(this.routeName);
     const backendModel = this.modelFor('vault.cluster.settings.configure-secret-backend');
-    const modelType = `${backendModel.get('type')}-config`;
+    const type = backendModel.get('type');
+    let modelType;
+    if (type === 'pki') {
```

Comment:
not ideal, but this route was not setup to handle folders. It assumed all models would be in the root models directory, which IMO is a little unmanageable or at least hard to view if you don't implement a sub-directory structure e.g. models/pki/model-name.

Reply:
I think we should agree upon a directory structure going forward or follow the recommended ember conventions to avoid having to handle exceptions like this when dynamically loading things.